shell/src/main/java/alluxio/cli/fs/command/DistributedLoadCommand.java

```diff
@@ -78,7 +78,8 @@
           .hasArg(true)
           .numberOfArgs(1)
           .argName("hosts")
-          .desc("A list of worker hosts separated by comma")
+          .desc("A list of worker hosts separated by comma, and add ! before host"
```

Comment:
I looked into the select worker logics, it's a little complicated. 
Basically, 
if nothing is specified in workers, we choose all.
If a worker is specified, it's choosen
if a worker is specified in excluded list, it will not be chosen for sure.

Then I would recommend adding the excludedHosts option and the excludedLocality option, which should be separated from the current hosts, locality options. Add parameter description saying that when hosts is empty, all hosts will be selected unless explicitly excluded by setting excludedHosts. hosts and excludedHosts should not be set together. Same for locality. 

The example ` [--hosts <host1,host2,...,hostN>] [--locality <locality1,locality2,...,localityN>]` is not a typical use case.

Reply:
Those changes are recommended for user usability and understability

Reply:
Thansk, I will modify code in the next commit, and I see it have <--host-file> option, should I set <--excluded-host-file>?

Reply:
Sounds good, we can also add that option.